spec/brainstem/presenter_collection_spec.rb

```diff
@@ -506,13 +506,13 @@ class MyException < Exception; end
       context "with include_params" do
         it "passes the params into the filter block" do
           WorkspacePresenter.filter :filter_with_param, :include_params => true do |scope, option, params|
```

Comment:
Should we have a second filter as well and make request that includes both, and show that the one with `include_params` sees the other param?